bin/data-get.js

```diff
@@ -40,7 +41,30 @@ const run = async () => {
     const parsedIdentifier = await parseDatasetIdentifier(identifier)
     const itIsDataset = isDataset(identifier)
     const githubDataset = parsedIdentifier.type === 'github' && parsedIdentifier.name.slice((parsedIdentifier.name.lastIndexOf('.') - 1 >>> 0) + 2) === ''
-    if (itIsDataset || parsedIdentifier.type === "datahub" || githubDataset) {
+
+    if(parsedIdentifier.type === "datahub"){
+      /**
+      https://github.com/datahq/datahub-qa/issues/86
+      For datasets from datahub we will get zipped version and unzip it.
+        - less traffic
+        - zipped version has a fancy file structure
+      */
+
+      // get zipped dataset archive
+      const dataset = await Dataset.load(identifier)
+      const zipped_dataset_resource = dataset.resources.filter(res => res.path.endsWith('.zip'))[0]
```

Comment:
how about `find` instead of `filter` so you wouldn't need to select 1st element and it stops after finding the first satisfying element:

```
const zipped_dataset_resource = dataset.resources.find(res => res.path.endsWith('.zip'))
```

Reply:
```
const zipped_dataset_url  = dataset.resources.find(res => res.path.endsWith('.zip')).path
```
Ok will do